### PR TITLE
Update plugin metadata

### PR DESCRIPTION
**Pull request by dstansby**
Fixes https://github.com/matplotlib/napari-matplotlib/issues/126

### CODE REVIEW

File: .napari/DESCRIPTION.md
Comment (dstansby) [inline .napari/DESCRIPTION.md:1]: I removed this because napari hub pulls the description form PyPI by deafult, so this is not needed.

**Review by ruaridhg — APPROVED** [review]
LGTM

File: .napari/DESCRIPTION.md
Comment (samcunliffe) [reply .napari/DESCRIPTION.md:1]: I'm all for minimising duplication but we get:

Screenshot 2023-06-01 at 14 49 17

Could you symlink it to README?

Comment (dstansby) [reply .napari/DESCRIPTION.md:1]: Not sure what's going on there, it shows up fine on the actual released version 🤔 https://www.napari-hub.org/plugins/napari-matplotlib

**Comment by dstansby** [comment]
In order to avoid faffing around to work out what's going on with the preview build, I'm going to merge this, and then double check the description appears correctly when I do a new release later.

**Pull request merged**